codes/classical/q-ary_digits/two_weight.yml

```diff
@@ -12,6 +12,9 @@ name: 'Two-weight code'
 
 description: 'Stub.'
 
+notes:
+  - ‘A table of binary constant weight codes of various lengths \(n\), less than or equal to twenty-eight, and examples of their construction by Brouwer \cite{manual:{A. E. Brouwer and Lames B. Shearer and N. I. A. Sloane and Warren and D. Smith, A new table of constant weight codes, IEEE Trans Inform Theory, 1990.}}, is preserved \cite{doi:10.1109/18.59932}{here}.'
```

Comment:
'A table of binary constant weight codes of various lengths \(n \leq 28\) \cite{doi:10.1109/18.59932}.'